.env

```diff
@@ -0,0 +1,2 @@
+KODI_HOME=tests/home
```

Comment:
Please don't include this file. I understand you use it for local development, but it should not be in the sources.

Reply:
You are right. Can I'll add it to .gitignore then.